src/dashboard/src/components/access/views.py

```diff
@@ -94,10 +94,17 @@ def all_records(client, request, system=''):
     search_pattern = request.GET.get('title', '')
     identifier = request.GET.get('identifier', '')
 
-    return helpers.json_response(client.find_collections(search_pattern=search_pattern,
-                                                         identifier=identifier,
-                                                         page=page,
-                                                         page_size=page_size))
+    records = client.find_collections(
+        search_pattern=search_pattern,
+        identifier=identifier,
+        page=page,
+        page_size=page_size,
+    )
+
+    for r in records:
+        r = _get_sip_arrange_children(r, system)
```

Comment:
Does this actually do anything? It's just reassigning the loop variable, not changing the entry in `records`, right?